core/src/main/java/org/mule/el/context/FlowVariableMapContext.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.el.context;
+
+import org.mule.api.MuleEvent;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class FlowVariableMapContext extends AbstractMapContext<String, Object>
```

Comment:
As I didn't know about AbstractMapContext, I took a look and seems to me that MessagePropertyMapContext should be updated as it can manage only outbound properties now.